### PR TITLE
Small amounts of cleanup

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Fixing the recommended cleanup suggestions in this file.

**Comment by CyrusNajmabadi** [comment]
tagging @jcouv

File: src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs [hunk 1]
```diff
@@ -1865,19 +1865,7 @@ private static void GetAccessibilityAndModifiers(SyntaxTokenList modifierList, o
...
                     case SyntaxKind.ProtectedKeyword:
-                        if (accessibility == Accessibility.Private)
-                        {
-                            accessibility = Accessibility.ProtectedAndInternal;
-                        }
-                        else if (accessibility == Accessibility.Internal)
-                        {
-                            accessibility = Accessibility.ProtectedOrInternal;
-                        }
-
```
Comment (sharwell) [inline src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs]: ❗️ This should say `else`, right?

Comment (CyrusNajmabadi) [reply src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs]: i could make that change as well :)  but this was just a behavior-preserving change :)

Comment (CyrusNajmabadi) [reply src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs]: #resolved

**Comment by CyrusNajmabadi** [comment]
@sharwell Fixed up your concern.  Can you merge in?

**Comment by sharwell** [comment]
@CyrusNajmabadi Can you amend e624611 to include 668cb43, and drop 08048ae and ff6edd7?

**Comment by CyrusNajmabadi** [comment]
@sharwell We can just squash this entire PR if you want.

**Comment by CyrusNajmabadi** [comment]
@sharwell Are you ok with this going in?

**Comment by sharwell** [comment]
I'm not a fan of 08048aed29e60b900addaf1cf2b18614b1ff0039 but I'll remove my blocking review.

**Comment by CyrusNajmabadi** [comment]
@sharwell Fixed up your concern.  Can you merge in?

**Comment by CyrusNajmabadi** [comment]
closing

**Comment by CyrusNajmabadi** [comment]
Opening.

**Comment by CyrusNajmabadi** [comment]
@sharwell can you merge this in?

**Comment by CyrusNajmabadi** [comment]
> @sharwell can you merge this in?

This is going on 4 months now...  thanks!

File: src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs [hunk 2]
```diff
@@ -1842,90 +1842,71 @@ private static void GetAccessibilityAndModifiers(SyntaxTokenList modifierList, o
...
-                        {
-                            accessibility = Accessibility.Protected;
-                        }
-
+                        accessibility = accessibility == Accessibility.Private
+                            ? Accessibility.ProtectedAndInternal
+                            : accessibility == Accessibility.Internal
+                                ? Accessibility.ProtectedOrInternal
+                                : Accessibility.Protected;
```
Comment (sharwell) [inline src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs]: 💡 `switch` expressions finally offer a clean alternative to multiply-nested ternary

**Comment by CyrusNajmabadi** [comment]
@sharwell can we merge this in?

**Comment by sharwell** [comment]
Yep merged!

**Pull request merged**